runtime/integration-tests/src/integration_tests.rs

```diff
@@ -472,7 +479,10 @@ fn liquidate_cdp() {
 				AccountId::from(ALICE),
 				10 * dollar(RELAY_CHAIN_CURRENCY),
 				50_000 * dollar(USD_CURRENCY),
+				#[cfg(feature = "with-mandala-runtime")]
 				LiquidationStrategy::Auction,
+				#[cfg(feature = "with-karura-runtime")]
+				LiquidationStrategy::Exchange,
```

Comment:
this make it testing two different thing. better to update whatever makes this different than updating assertions.
@syan095 have identified why this is different.